### ISSUE TITLE
Mapping flows in ggplot

### ISSUE THREAD

**Issue by RPanczak** [ropensci/ozunconf18#18]
It won't be a surprise for you to hear that #Rstats is becoming more and more capable in working with spatial data :world_map: 
With awesome tools like `sf` and `tmap` working already very well there is very little need for standalone Geographic Information Systems (GIS) software and it's great to have all your munging, analyses and cartography all in one place. 

One thing that I think is lagging behind in this domain is visualization of geographical flows. If you ignore geography - there are some interesting tools for visualization like Sankey diagrams or `RCircos`.

Some seriously pretty #Rstats graphs were generated already a while ago ([example](http://spatialanalysis.co.uk/2012/06/mapping-worlds-biggest-airlines/), and one [more](http://spatial.ly/2016/10/mapping-historic-tracks-ggplot2/)). My feeling though is that they were more focused on the prettiness factor than what cartography and visualization would advise to do. 

[Recent paper](https://www.tandfonline.com/doi/abs/10.1080/15230406.2016.1262280) formalized what good flow maps should look like and some attempts have been done in GIS to [implement that](https://anitagraser.com/2016/12/18/details-of-good-flow-maps/). 

I think it'd be super cool to have a `ggplot` way of visualizing flows like that. There are some neat data from Austrlian Census that would make a perfect testbed for the project looking at commuting to work or difference between _de facto_ and _de jure_ populations on various [scales of geography](http://www.abs.gov.au/websitedbs/D3310114.nsf/home/Australian+Statistical+Geography+Standard+(ASGS)).

**Comment by adam-gruer** [comment]
Paul Murrell’s vwlines package may be useful.

**Comment by adam-gruer** [comment]
And maybe ggnetwork or ggraph. Reading through the resources above put me in mind of network graphs with nodes and edges

**Comment by mrjoh3** [comment]
The visualising flows issue #23 mentioned a package processannimateR which appears to have a leaflet maps option.

I like the idea of visualizing with gganimate . I am interested in visualising the global trade in waste plastic. This is reasonably straight forward as a snapshot. But the trade is incredibly dynamic so you really need to show how everything changed through time.

**Comment by RPanczak** [comment]
It's happening! :smile_cat: https://github.com/ropenscilabs/geoflowr partly mering with #23 come and see us in main room if you are interested in some pretty (hopefully animated at some stage) maps.

**Comment by RPanczak** [comment]
Indeed. But weare also aiming for animated component.. Either `gganimate` or `mapmate` https://leonawicz.github.io/mapmate/index.html

**Comment by SymbolixAU** [comment]
Ok cool. I'm trying to add animation capability into mapdeck too. A lot still to do though
